stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -276,6 +277,9 @@ public class MessageListView : ConstraintLayout {
 
     private var deletedMessageListItemPredicate: MessageListItemPredicate =
         DeletedMessageListItemPredicate.VisibleToEveryone
+
+    internal var deletedMessageListItemPredicateLiveData: MutableLiveData<MessageListItemPredicate?> = MutableLiveData(null)
```

Comment:
Made this null so we can ignore the first emission.
Otherwise you could update the same setting in the VM and the initial emission of this value would overturn that one.